cron/disabled/expiring-cards.js

```diff
@@ -5,14 +5,14 @@ import logger from '../../server/lib/logger';
 import models from '../../server/models';
 import * as libPayments from '../../server/lib/payments';
 
-// Only run on the first of the month
+// Run on the 7th and 21st of the month
 const today = new Date();
 const date = today.getDate();
 const month = today.getMonth() + 1;
 const year = today.getFullYear();
 
-if (process.env.NODE_ENV === 'production' && date !== 1 && !process.env.OFFCYCLE) {
-  console.log('NODE_ENV is production and today is not the first of month, script aborted!');
+if (process.env.NODE_ENV === 'production' && (date !== 7 || date !== 21) && !process.env.OFFCYCLE) {
```

Comment:
Seems to me that this condition will never be falsy, for example with date=21:

```
// true                                   && (true       || false      ) && true
if (process.env.NODE_ENV === 'production' && (date !== 7 || date !== 21) && !process.env.OFFCYCLE) {
```

Reply:
Apologies - the || here should be an &&, correct?